Sources/SQLKit/Query/SQLUnion.swift

```diff
@@ -0,0 +1,22 @@
+public struct SQLUnion: SQLExpression {
+    public let args: [SQLExpression]
+
+    public init(_ args: SQLExpression...) {
+        self.init(args)
+    }
+
+    public init(_ args: [SQLExpression]) {
+        self.args = args
+    }
+
+    public func serialize(to serializer: inout SQLSerializer) {
+        let groups = args.map(SQLGroupExpression.init)
+        guard let first = groups.first else { return }
+        first.serialize(to: &serializer)
+        for arg in groups.dropFirst() {
+            serializer.write(" UNION ")
```

Comment:
Just realised there's also `UNION ALL`, at least on Postgres. Not sure if it's universal across supported SQL flavours but I guess it couldn't hurt to support this via a flag defaulting to off.